elements/pl-integer-input/pl-integer-input.py

```diff
@@ -37,9 +37,23 @@ def prepare(element_html, data):
         if name in data['correct_answers']:
             raise Exception('duplicate correct_answers variable name: %s' % name)
         # Test conversion, but leave as string so proper value is shown on answer panel
-        if pl.string_to_integer(correct_answer, base) is None:
+        a_tru_parsed = pl.string_to_integer(correct_answer, base)
+        if a_tru_parsed is None:
             raise Exception('correct answer is not a valid input: %s' % name)
+        if a_tru_parsed > 2**53 - 1 or a_tru_parsed < -((2**53) - 1):
+            raise Exception('correct answer must be between -9007199254740991 and +9007199254740991 (that is, between -(2^53 - 1) and +(2^53 - 1)).')
```

Comment:
This is probably not needed since the same test is also done in lines 51-56.

Reply:
But the 51-56 tests don't run if you get the answer from the html

Reply:
You update `correct_answers` in line 45, why wouldn't the tests run if it comes from HTML?

Reply:
wouldn't the return on line 46 make it so it never reaches the other section?

Reply:
oh, i could just remove the return and have a check if the answer has been loaded and if not load it?,